spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/QuantizedTensor.scala

```diff
@@ -0,0 +1,356 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.tensor
+
+import breeze.linalg.{DenseMatrix, DenseVector}
+import com.intel.analytics.bigdl.bigquant.BigQuant
+import com.intel.analytics.bigdl.nn.quantized._
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.Table
+import java.nio.ByteBuffer
+import org.apache.spark.mllib.linalg
+import org.apache.spark.mllib.linalg.Matrix
+import scala.reflect.ClassTag
+
+@SerialVersionUID(- 1766499387282335147L)
+private[bigdl] class QuantizedTensor[@specialized(Float) T: ClassTag](
+  private[bigdl] var _size: Array[Int],
+  private[bigdl] var _stride: Array[Int],
+  var nDimension: Int)(implicit ev: TensorNumeric[T]) extends QuantTensorUnsupported[T] {
+  @transient private var desc = 0L
+  private var interStorage: Array[Byte] = null
+  private var setFromOther: Boolean = false
+
+  var maxOfRow: Array[T] = null
+  var minOfRow: Array[T] = null
+  var sumOfRow: Array[T] = null
+
+  var params: DescParams = _
+  var descType: DescType = _
+
+  private[bigdl] def setStorage(buffer: Array[Byte]): this.type = {
```

Comment:
The entire class is already private to bigdl, yes? And where is this method used?